src/main/resources/entity-types/finance/drv_fund_details.json5

```diff
@@ -0,0 +1,286 @@
+{
+  id: '4c63c7f5-1a28-4394-b401-98c02c31492d',
+  name: 'drv_fund_details',
+  private: false,
+  fromClause: 'src_fund as fnd',
+  columns: [
+    {
+      name: 'id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      isIdColumn: true,
+      valueGetter: "fnd.id"
+    },
+    {
+      name: '_version',
+      dataType: {
+        dataType: 'integerType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb ->> '_version'"
+    },
+    {
+      name: 'allocated_from_ids',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'rangedUUIDType',
+        },
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb ->> 'allocatedFromIds'"
+    },
+    {
+      name: 'allocated_to_ids',
+      dataType: {
+        dataType: 'arrayType',
+        itemDataType: {
+          dataType: 'rangedUUIDType',
+        },
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb ->> 'allocatedToIds'"
```

Comment:
This will give us a text object, but with non-object arrays we've been using text[]. I think we might need it to be a text[] for querying to work.